code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -195,17 +194,18 @@
 				var/obj/structure/spider/cocoon/C = new(cocoon_target.loc)
 				if(isliving(cocoon_target))
 					var/mob/living/L = cocoon_target
-					if(L.blood_volume && (L.stat != DEAD || !consumed_mobs[L.tag]) && !isipc(L)) //if they're not dead, you can consume them anyway
-						if(istype(L, /mob/living/carbon/human))
-							enriched_fed++
-						else
-							fed++
-						consumed_mobs[L.tag] = TRUE
+					if(L.stat != DEAD)
+						L.death() //If it's not already dead, we want it dead regardless of nourishment
+					if(L.blood_volume && !isipc(L)) //IPCs and bloodless mobs are not nourishing.
```

Comment:
Should probably check that blood volume is over a certain threshold just to be sure that the dead body doesn't absorb blood or get abused in some way

Reply:
Spiders have no way to do this abuse themselves since they can't inject corpses with blood, but it's not a bad failsafe idea
👍